workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/StandardGameObjectCreation/GameObjectCreationSystemHelper.cs

```diff
@@ -0,0 +1,18 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static class GameObjectCreationSystemHelper
```

Comment:
How about making this part of the `EntityGameObjectLinkerSystem` and renaming that system to something like `EntityGameObjectCreatorSystem`? That way we get rid of the staticness and don't need to pass in worlds anymore.

Reply:
Discussed offline, not possible due to code visibility and consistent with how we do things elsewhere